src/odemis/gui/main_xrc.py

```diff
@@ -3917,7 +3915,7 @@ def __init_resources():
                       </XRCED>
                     </object>
                     <option>1</option>
-                    <flag>wxALL|wxEXPAND|wxALIGN_BOTTOM|wxALIGN_CENTRE_VERTICAL</flag>
```

Comment:
Why is the align bottom also removed here?

Reply:
All ALIGN_* are incompatible with EXPAND.